codecov.yml

```diff
@@ -1,17 +1,18 @@
 flag_management:
   default_rules: # the rules that will be followed for any flag added, generally
     carryforward: false
+    after_n_builds: 5 # should correspond to the number of coverage reports we expect
```

Comment:
Delays the status-check reporting until this number of reports have been uploaded.

This prevents unit tests finishing first and causing `codecov` to complain about missing coverage that will be reported once integration tests finish.